apps/apps.h

```diff
@@ -192,31 +192,43 @@ void wait_for_async(SSL *s);
+        { "policy", OPT_V_POLICY, 's', "adds policy to the acceptable policy set"}, \
+        { "purpose", OPT_V_PURPOSE, 's', \
+            "Set the acceptable purpose of the certificate chain"}, \
+        { "verify_name", OPT_V_VERIFY_NAME, 's', "verify name"}, \
+        { "verify_depth", OPT_V_VERIFY_DEPTH, 'p', \
+            "Limit the maximum depth of the certificate chain"}, \
+        { "attime", OPT_V_ATTIME, 'M', "Set the verification time" }, \
+        { "verify_hostname", OPT_V_VERIFY_HOSTNAME, 's', \
+            "check peer certificate matches \"host\"" }, \
+        { "verify_email", OPT_V_VERIFY_EMAIL, 's', \
+            "check peer certificate matches \"email\"" }, \
+        { "verify_ip", OPT_V_VERIFY_IP, 's', \
+            "check peer certificate matches \"ipaddr\"" }, \
+        { "ignore_critical", OPT_V_IGNORE_CRITICAL, '-', \
+            "Disable critical extension checking"}, \
+        { "issuer_checks", OPT_V_ISSUER_CHECKS, '-', \
+            "Enable debugging of certificate issuer checks"}, \
         { "crl_check", OPT_V_CRL_CHECK, '-', "Check that peer cert has not been revoked" }, \
         { "crl_check_all", OPT_V_CRL_CHECK_ALL, '-', "Also check all certs in the chain" }, \
-        { "policy_check", OPT_V_POLICY_CHECK, '-' }, \
-        { "explicit_policy", OPT_V_EXPLICIT_POLICY, '-' }, \
-        { "inhibit_any", OPT_V_INHIBIT_ANY, '-' }, \
-        { "inhibit_map", OPT_V_INHIBIT_MAP, '-' }, \
-        { "x509_strict", OPT_V_X509_STRICT, '-' }, \
+        { "policy_check", OPT_V_POLICY_CHECK, '-', "Enable certificate policy checking"}, \
+        { "explicit_policy", OPT_V_EXPLICIT_POLICY, '-', "Set the \"require explicit policy\""}, \
+        { "inhibit_any", OPT_V_INHIBIT_ANY, '-', "Set the \"inhibit any policy\"\""}, \
+        { "inhibit_map", OPT_V_INHIBIT_MAP, '-', "Set the \"inhibit policy mapping\"" }, \
+        { "x509_strict", OPT_V_X509_STRICT, '-', \
+            "Strictly apply X509 rules in verification"}, \
         { "extended_crl", OPT_V_EXTENDED_CRL, '-' }, \
-        { "use_deltas", OPT_V_USE_DELTAS, '-' }, \
-        { "policy_print", OPT_V_POLICY_PRINT, '-' }, \
-        { "check_ss_sig", OPT_V_CHECK_SS_SIG, '-' }, \
-        { "trusted_first", OPT_V_TRUSTED_FIRST, '-', "Use locally-trusted CA's first in building chain" }, \
-        { "suiteB_128_only", OPT_V_SUITEB_128_ONLY, '-' }, \
-        { "suiteB_128", OPT_V_SUITEB_128, '-' }, \
-        { "suiteB_192", OPT_V_SUITEB_192, '-' }, \
+        { "use_deltas", OPT_V_USE_DELTAS, '-', \
+            "Enable indirect CRLs and CRLs signed by different keys"}, \
+        { "policy_print", OPT_V_POLICY_PRINT, '-', "Notify callback that policy is OK"}, \
+        { "check_ss_sig", OPT_V_CHECK_SS_SIG, '-', \
+            "Enable checking of the root CA self signed certificate signature"}, \
+        { "trusted_first", OPT_V_TRUSTED_FIRST, '-', \
+            "Use locally-trusted CA's first in building chain (enabled by default)" }, \
```

Comment:
man verify(1) says this:

 -trusted_first
           When constructing the certificate chain, use the trusted certificates specified via -CAfile, -CApath or -trusted before any certificates specified
           via -untrusted.  This can be useful in environments with Bridge or Cross-Certified CAs.  **As of OpenSSL 1.1.0 this option is on by default and cannot
           be disabled.**

Hence I thought it is already implemented it that way. Not removed by self because I have not tested myself :)
If it is always enabled, probably the option can be removed. Thanks.